client/my-sites/stats/activity-log-banner/success-banner.jsx

```diff
@@ -45,8 +48,12 @@ class SuccessBanner extends PureComponent {
 					'We successfully restored your site back to %s!',
 					{ args: moment( timestamp ).format( 'LLLL' ) }
 				) }</p>
-				<Button primary>
-					{ /* FIXME: Link to site */ translate( 'View site' ) }
+				<Button
+					href={ siteUrl }
+					primary
+					rel="noopener noreferrer"
```

Comment:
Do we need `rel="noopener noreferrer"` when we don't have a `target="_blank"`?

Perhaps we should add it, since this button opens an external website? But in that case, `rel="noopener noreferrer"` will not be necessary, as the `Button` component will add it if we have specified a `target`.